Neos.Neos/Classes/Controller/Module/Administration/SitesController.php

```diff
@@ -644,7 +644,7 @@ public function deactivateDomainAction(Domain $domain)
      * @param string $format The format to use for the redirect URI
      * @return void
      */
-    protected function unsetLastVisitedNodeAndRedirect(
+    protected function redirectToAction(
```

Comment:
it seems we can instead always just call `$this->redirect` instead and remove this `redirectToAction` completely?

Reply:
haha 3 seconds :P 0e7f1dfe85b303500569fbafa93061bd964bef92